step-templates/iis-app-create.json

```diff
@@ -79,13 +73,85 @@
+      "Label": "Enable Anonymous authentication",
+      "HelpText": "Whether IIS should allow anonymous authentication",
+      "DefaultValue": "False",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Name": "EnableBasic",
+      "Label": "Enable Basic authentication",
+      "HelpText": "Whether IIS should allow basic authentication with a 401 challenge.",
+      "DefaultValue": "False",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Name": "EnableWindows",
+      "Label": "Enable Windows authentication",
+      "HelpText": "Whether IIS should allow integrated Windows authentication with a 401 challenge.",
+      "DefaultValue": "False",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Name": "ApplicationPoolName",
+      "Label": "Application pool",
+      "HelpText": "The name of the application pool that the application will run under. The application pool must already exist.",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ApplicationPoolFrameworkVersion",
+      "Label": "Application Pool .NET CLR Version",
+      "HelpText": "The version of the .NET common language runtime that this application pool will use. Choose v2.0 for applications built against .NET 2.0, 3.0 or 3.5. Choose v4.0 for .NET 4.0 or 4.5",
+      "DefaultValue": "V4.0",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "V4.0\nV2.0\nNo Managed Code"
+      }
+    },
+    {
+      "Name": "ApplicationPoolIdentityType",
+      "Label": "Application Pool Identity",
+      "HelpText": "Which built-in account will the application pool run under.",
+      "DefaultValue": "ApplicationPoolIdentity",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "ApplicationPoolIdentity|Application Pool Identity\nLocalService|Local Service\nLocalSystem|Local System\nNetworkService|Network Service\nSpecificUser|Custom user ..."
+      }
+    },
+    {
+      "Name": "ApplicationPoolUsername",
+      "Label": "Application Pool Username",
+      "HelpText": "The Windows/domain account of the custom user that the application pool will run under. Example: YOURDOMAIN\\\\YourAccount. You will need to ensure that this user has permissions to run as an application pool.",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ApplicationPoolPassword",
+      "Label": "Application Pool Password",
+      "HelpText": "The password for the custom account given above.",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
     }
   ],
-  "LastModifiedBy": "ekrapfl",
```

Comment:
If you add this back in and modify them with your GitHub username, it will be shown on the library site